recipes/pangolin/meta.yaml

```diff
@@ -19,23 +19,23 @@ requirements:
     - pip
   run:
     - python >=3.7
-    - biopython ==1.74
+    - biopython >=1.74
     - pandas >=1.0.1
-    - tabulate
     - joblib >=0.11
-    - scikit-learn <1.3.0
+    - scikit-learn >=0.23.1
     - pulp >=2
     - minimap2 >=2.16
-    - snakemake-minimal >=5.13,<=7.30.1
+    - snakemake-minimal >=5.13,!=7.30.1
     - gofasta
-    - usher >=0.6.2
+    - usher >=0.5.4
```

Comment:
see https://github.com/cov-lineages/pangolin/issues/500 for why usher is pinned to >=0.6.2